tools/twix/src/panels/enum_plot.rs

```diff
@@ -133,16 +138,18 @@ impl EnumPlotPanel {
             .name(&name)
             .stroke(Stroke::new(BORDER_WIDTH, color)),
         );
+
+        let mut text_x = start + 0.05;
+        if viewport_left_edge > text_x
+            && end > (viewport_left_edge + (min_boxsize_text * MIN_SEGMENT_RELATION_FOR_TEXT))
+        {
+            text_x = viewport_left_edge + 0.05;
+        }
```

Comment:
Rewrite without `mut`
```suggestion
        let text_x = if start < viewport_left_edge && viewport_left_edge < end {
            viewport_left_edge + 0.05
        } else {
            start + 0.05
        };
```